README.md

```diff
@@ -27,8 +36,29 @@ In order to run GraphSpace, please install postgreSQL and both the Python runtim
 3. Create a virtual environment for the project: `virtualenv venv`
 4. Start using the virtual environment: `source venv/bin/activate`
 5. Install graphspace: `sh install.sh`
-6. Finally, start the GraphSpace server: `python manage.py runserver --settings=graphspace.settings.local`
-7. Visit `http://localhost:8080` and enjoy using GraphSpace!
+   
+   _Note : While installing psycopg2, you might encounter 
+   ```Error: You need to install postgresql-server-dev-X.Y for building a server-side extension or libpq-dev for building a client-side application.```
+   To fix it run the following commands ([More](https://stackoverflow.com/a/28254860/4646197)):_
+   ```
+   sudo apt-get install libpq-dev python-dev
+   pip install psycopg2
+   ```
+6. In `/graphspace/settings/local.py` file, change the postgres user credentials:
```

Comment:
asgi.py depends on production.py initially. Should we mention that file in the README as well?